### PR TITLE
Updates recipes for ISIS cmake changes

### PR DESCRIPTION
**Pull request by chrisryancombs**

### CODE REVIEW

File: recipies/tnt/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
   name: tnt
-  version: {{ package_versions.tnt }}
+  version: 126
 
 source:
-  fn: tnt_{{ package_versions.bullet }}.zip
-  url: http://math.nist.gov/tnt/tnt_{{ package_versions.tnt }}.zip
-
+  fn: master.zip
+  url: https://github.com/chrisryancombs/TNT/archive/master.zip
```
Comment (jlaura) [inline recipies/tnt/meta.yaml]: Can you PR the TNT stuff into: https://github.com/USGS-Astrogeology/TNT and then update this?

File: recipies/isis/build.sh [hunk 2]
```diff
@@ -1,3 +1,6 @@
-
-cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug  -DJP2KFLAG=OFF -DbuildTests=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX isis
+mkdir build
+cd build
+ls /usgs/apps/kakadu/v7_9_1-01762L/managed/all_includes/
+cmake -GNinja -DJP2KFLAG=ON -Dpybindings=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX -Disis3Data=/usgs/cpkgs/isis3/data -Disis3TestData=/usgs/cpkgs/isis3/testData ../isis
```
[the remark this answers is not in the document]
Comment (chrisryancombs) [reply recipies/isis/build.sh]: Most of this is documented here : https://github.com/USGS-Astrogeology/ISIS3/wiki/Managing-Custom-ISIS3-Dependencies-(WIP)

File: recipies/tnt/build.sh [hunk 3]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 mkdir -p ${PREFIX}/include/tnt/
-cp *.h ${PREFIX}/include/tnt/
+cp tnt/*.h ${PREFIX}/include/tnt/
```
[the remark this answers is not in the document]
Comment (chrisryancombs) [reply recipies/tnt/build.sh]: TNT is still in

File: recipies/isis/meta.yaml [hunk 4]
```diff
@@ -14,17 +14,19 @@ build:
...
     - qwt=6.1.3
-    - pyqt==5.6.0
+    # - pyqt==5.6.0
     - sip==4.18
     - mysql==5.7.20
     - ninja==1.7.2=0
-    - patchelf==0.9
+    - xorg-libsm
+    - xorg-libxi
+    - xorg-libice
```
[the remark this answers is not in the document]
Comment (chrisryancombs) [reply recipies/isis/meta.yaml]: Not sure. We will stick with the shotgun method for now and remove redundant dependancies later

**Pull request merged**